web_server/posts/models.py

```diff
@@ -11,7 +11,8 @@ class Category(models.Model):
     name = models.CharField(max_length=256, null=False, unique=True)
 
     def __str__(self):
-        return f'{self.name}'
+        return "{}".format(self.name)
```

Comment:
Any particular reason you do not want to use f strings? They are succinct, and supported by the version of python we are supposed to target.

Reply:
if you require everyone's Python version to be at least 3.6 +, I am fine with using f string.